packages/paste-website/src/pages/experiences/artificial-intelligence/index.mdx

```diff
@@ -0,0 +1,315 @@
+export const meta = {
+  title: 'Artificial Intelligence',
+  description:
+    'AI experiences help our customers meet their goals better and faster through predictions, recommendations, and insights.',
+  slug: '/experiences/artificial-intelligence/',
+};
+
+import { SidebarCategoryRoutes } from "../../../constants";
+import {ResponsiveImage} from '../../../components/ResponsiveImage';
+import {Blockquote} from '../../../components/Blockquote';
+import {Anchor} from '@twilio-paste/anchor';
+import {ArtificialIntelligenceIcon} from '@twilio-paste/icons/esm/ArtificialIntelligenceIcon';
+import {Button} from '@twilio-paste/button';
+import {ButtonGroup} from '@twilio-paste/button-group';
+import {Badge} from '@twilio-paste/badge';
+import {Box} from '@twilio-paste/box';
+import {ChatComposerContainer, ChatComposer, ChatComposerActionGroup} from '@twilio-paste/chat-composer';
+import {SendIcon} from '@twilio-paste/icons/esm/SendIcon';
+import {Label} from '@twilio-paste/label';
+import {TextArea} from '@twilio-paste/textarea';
+import {HelpText} from '@twilio-paste/help-text';
+import {Callout, CalloutHeading, CalloutText} from '@twilio-paste/callout';
+import {Heading} from '@twilio-paste/heading';
+import {Popover, PopoverContainer, PopoverButton} from '@twilio-paste/popover';
+import {NewIcon} from '@twilio-paste/icons/esm/NewIcon';
+import { ProductMessagingIcon } from "@twilio-paste/icons/esm/ProductMessagingIcon";
+import { ProductSendGridIcon } from "@twilio-paste/icons/esm/ProductSendGridIcon";
+import { ProductCLIIcon } from "@twilio-paste/icons/esm/ProductCLIIcon";
+import {Paragraph} from '@twilio-paste/paragraph';
+import DefaultLayout from '../../../layouts/DefaultLayout';
+import {getFeature, getNavigationData} from '../../../utils/api';
+
+export default DefaultLayout;
+
+export const getStaticProps = async () => {
+  const navigationData = await getNavigationData();
+  const feature = await getFeature('Artificial Intelligence');
+  return {
+    props: {
+      data: {
+        ...feature,
+      },
+      navigationData,
+    },
+  };
+};
+
+
+<NormalizedPatternHeader description={meta.description} name={meta.title} data={props.data} categoryRoute={SidebarCategoryRoutes.EXPERIENCES} />
+
+
+<contentwrapper>
+
+<PageAside data={mdxHeadings} />
+
+<content>
+
```

Comment:
suggestion: add the empty state example up here, too, before we hit people with a wall of text